packages/create-remix/index.ts

```diff
@@ -43,7 +45,8 @@ async function createRemix(argv: string[]) {
   let steps = [
     introStep,
     projectNameStep,
-    templateStep,
+    copyTemplateToTempDirStep,
+    copyTempDirToAppDirStep,
```

Comment:
In order to do the file level comparison, we now extract  into a temp directory and then compare the temp dir to the destination dir.